app/views/smart_proxies/plugins/_realm.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <h4><%= _('Realm Details') %></h4>
```

Comment:
I think we can drop the word "details", when having multiple features the context is clear imho:
![details](https://cloud.githubusercontent.com/assets/20167/12009785/9ca36c2a-ac90-11e5-9733-dea39ffc14c4.png)